apps/passport-client/src/dispatch.ts

```diff
@@ -436,6 +471,31 @@ async function loadFromSync(
   }
 }
 
+// Update `self` and `encryptionKey` in-memory fields from their saved values in localStorage
+async function updateStateFromLocalStorage(update: ZuUpdate) {
+  const self = loadSelf();
+  const encryptionKey = loadEncryptionKey();
+  return update({
+    self,
+    encryptionKey
+  });
+}
+
+async function saveNewPasswordAndBroadcast(
+  newEncryptionKey: string,
+  newSalt: string,
+  state: AppState,
+  update: ZuUpdate
+) {
+  const newSelf = { ...state.self, salt: newSalt };
+  saveSelf(newSelf);
```

Comment:
this doesn't save the encryption key to local storage, though it does save the salt.

Reply:
the encryption key is saved in `onChangePassword`, but I'll move it here for clarity.